contrib/wheel_add_ucx_plugins.py

```
@@ -261,6 +305,21 @@ def add_plugins(wheel_path, sys_plugins_dir, install_dirname):
                     ):
                         raise RuntimeError(f"Library {libname} not loaded by {fpath}")
 
+    # Replace inter-plugin DT_NEEDED entries that reference removed symlink
+    # names (e.g. libuct_ib.so.0) with the kept real file name
+    # (e.g. libuct_ib.so.0.0.0).
+    if dedup_map:
+        for fname in copied_files:
+            if os.path.isfile(fname) and ".so" in fname:
+                for old_name, new_name in dedup_map.items():
+                    ret = os.system(
```

Ruff (0.15.10)

[error] Starting a process with a shell, possible injection detected

(S605)